src/components/Header.vue

```diff
@@ -26,7 +26,6 @@
           <input type="text" :value="shareLink" readonly class="form-control form-control-sm mr-2" id="link">
         </div>
         <button :disabled="shareLink ? true : false" @click="nodeSubmit" class="btn btn-outline-dark btn-sm mr-5">SHARE</button>
```

Comment:
`:disabled` should be boolean in your data, maybe you could change this variable to `isDisabled` and stick with shareLink variable as your string data